app/questionnaire/user_journey_manager.py

```diff
@@ -0,0 +1,85 @@
+import logging
```

Comment:
This functionality should be implemented along the same lines as metadata in data_model/questionnaire_store.